WalletWasabi.Gui/Config.cs

```diff
@@ -84,6 +86,24 @@ public class Config : ConfigBase
 		[JsonConverter(typeof(EndPointJsonConverter), Constants.DefaultRegTestBitcoinP2pPort)]
 		public EndPoint RegTestBitcoinP2pEndPoint { get; internal set; } = new IPEndPoint(IPAddress.Loopback, Constants.DefaultRegTestBitcoinP2pPort);
 
+		[DefaultValue(false)]
+		[JsonProperty(PropertyName = "JsonRpcServerEnabled", DefaultValueHandling = DefaultValueHandling.Populate)]
+		public bool JsonRpcServerEnabled { get; internal set; }
+
+		[DefaultValue("")]
+		[JsonProperty(PropertyName = "JsonRpcUser", DefaultValueHandling = DefaultValueHandling.Populate)]
+		public string JsonRpcUser { get; internal set; }
+
+		[DefaultValue("")]
+		[JsonProperty(PropertyName = "JsonRpcPassword", DefaultValueHandling = DefaultValueHandling.Populate)]
+		public string JsonRpcPassword { get; internal set; }
+
+		[JsonProperty(PropertyName = "JsonRpcServerPrefixes")]
```

Comment:
Shouldn't it be hosts? Prefixes doesnt tell anything to me.

Reply:
It is true, `Prefixes` is confusing. I just wanted to use the same wording used in the `HttpListener` what probably is not the best. However, `host` is even worse because the prefix is an url pattern like `http://*.localhost:8080/`. Bull Bitcoin team is using `http://*:18099/` because in that way they can map the internal port to one external port. So, instead of `Prefixes` it could be `Urls`. What do you think?